src/IW/Server/Issue.hs

```diff
@@ -0,0 +1,83 @@
+module IW.Server.Issue
+       ( -- * API
+         IssueAPI
+       , issueServer
+
+         -- * Handlers
+       , issueByIdHandler
+       , issuesHandler
+       ) where
+
+import IW.App.Error (WithError)
+import IW.Core.Issue (Issue (..))
+import IW.Db (WithDb, getIssues, getIssueById, getIssuesByLabel)
+import IW.Server.Types (AppServer, ToApi)
+
+data Sort = Id | Title
+  deriving (Generic, Show, Eq)
+
+instance FromHttpApiData Sort where
+    parseQueryParam :: Text -> Either Text Sort
+    parseQueryParam "id"    = Right Id
+    parseQueryParam "title" = Right Title
+    parseQueryParam _       = Left "Error parsing sort value"
+
+data IssueSite route = IssueSite
+    { -- | Fetch issue by issue ID
+      issueByIdRoute :: route
+        :- "issues"
+        :> Capture "id" Int
+        :> Get '[JSON] Issue
+    , issuesRoute :: route
+        :- "issues"
+        :> QueryParam "sort" Sort
```

Comment:
I'm not sure if it is ok to have this as a query parameter. I think it is not an optimal solution and could be too slow. What about the possibility of having sorting on the frontend side?

Reply:
@vrom911 I agree. Calling backend for sorting is a bit overkill and will result in slow performance. I propose to think carefully about what we want to do on frontend and backend. Here are my thoughts:

1. By default frontend initially frontend asks for 100 latest created issues.
2. Sorting is done purely on frontend. I think for now it makes sense to sort only by time of creation.
3. We need to discuss how we want to filter issues. We might want to filter by:
    * Zero or more labels
    * Zero or more categories

Filtering should be done on the backend, because if frontend knows only 100 recent all issues, it can't know 100 recent issues with the label `help wanted`

Reply:
Sounds good. I thought the same, and wanted to try and do sorting at the database level with SQL, but didn't know how to implement it. I was thinking of creating a whole bunch of different SQL queries for each type of sorting situation, but that wouldn't be fun lol